### DISCUSSION TITLE
Porting to new platforms

### DISCUSSION THREAD

**Discussion by different55** [nesbox/TIC-80#1473] [Q&A]
Hypothetically, if I was wanting to port TIC-80 to a new platform, how far in over my head would I hypothetically be? And hypothetically where would I start?

**Comment by different55** [comment]
You know what, spending another 5 minutes looking into the minimum requirements for the TIC-80, I don't think a port would be workable anyway. Doesn't seem to be a Delete button so instead you get this lightning fast backtrack.

**Reply by Anrock** [reply]
I can delete this discussion if you wish.

**Reply by different55** [reply]
Nah, it'll be fine. Honestly probably worth it to have it somewhere search engines can index that the PSP wouldn't be able to handle everything that people are doing with the TIC-80, no matter how nicely the TIC-80's screen might map to the PSP's.

**Reply by ChildishGiant** [reply]
I'm hoping someone can port tic to the Playdate when it comes out, especially since we get a 1bpp  mode

**Comment by RobLoach** [comment]
Check out the build dir, along with the cmake definition files... https://github.com/nesbox/TIC-80/tree/master/build
